src/main/scala/EnsimePlugin.scala

```diff
@@ -553,14 +575,24 @@ case class EnsimeModule(
   runtimeJars: Set[File],
   testJars: Set[File],
   sourceJars: Set[File],
-  docJars: Set[File]
+  docJars: Set[File],
+  configurations: List[EnsimeConfiguration]
```

Comment:
it might be cleaner to create a different class that sits beside the module, rather than adding to it. e.g. the containing class would have `subprojects: Seq[EnsimeModule], projects: Seq[ThisNewThing]` so you wouldn't need to touch the old `EnsimeModule` definition or writer code at all.